addon/components/dropzone-widget/component.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+    layout,
+    session: Ember.inject.service(),
+    classNames: ['dropzone'],
+    didRender() {
+        this.buildUrl = this.get('buildUrl');
```

Comment:
Is there a need to do assignment for `buildUrl`? Using `this.get('buildUrl')` is equivalent